docs/dev/table/sql/create.md

```diff
@@ -210,6 +210,22 @@ The key and value of expression `key1=val1` should both be string literal. See d
 
 {% top %}
 
+## CREATE CATALOG
+
+{% highlight sql %}
+CREATE CATALOG catalog_name
+  WITH (key1=val1, key2=val2, ...)
+{% endhighlight %}
+
+Create a catalog with the given catalog properties. If a catalog with the same name already exists, an exception is thrown.
+
+**WITH OPTIONS**
+
+Catalog properties used to store extra information related to this catalog.
+The key and value of expression `key1=val1` should both be string literal.
```

Comment:
Are there any required property? Could I just write _CREATE CATALOG catalog_name_ or _CREATE CATALOG catalog_name WITH ()_?

Reply:
I think we should add a link to catalog page: https://ci.apache.org/projects/flink/flink-docs-master/dev/table/catalogs.html#using-sql-ddl
And would be better to add an example here.

Reply:
good idea. I'll add a link